### PR TITLE
Camera Based Scanner

### PR DESCRIPTION
**Pull request by Jahnavi-Jonnalagadda**

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -402,5 +409,5 @@ packages:
     source: hosted
     version: "4.5.1"
 sdks:
-  dart: ">=2.12.0-0.0 <3.0.0"
+  dart: ">=2.10.0-110 <2.11.0"
```
Comment (smaranjitghose) [inline pubspec.lock:432]: Update your dependencies please

Comment (Jahnavi-Jonnalagadda) [reply pubspec.lock:432]: Sure, will do

Comment (Jahnavi-Jonnalagadda) [reply pubspec.lock:432]: @smaranjitghose, should I raise a PR after updating the dependencies?

Comment (smaranjitghose) [reply pubspec.lock:432]: yes

Comment (Jahnavi-Jonnalagadda) [reply pubspec.lock:432]: I'll have to update my pubspec.lock file right?

Comment (smaranjitghose) [reply pubspec.lock:432]: yes

**Pull request merged**